src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+  public void doWork() {
+        System.out.println("Bulldozer starts working.");
+    }
+
+    @Override
+  public void stopWork() {
+        System.out.println("Bulldozer stops.");
+    }
```

Comment:
Fix formatting according to codestyle here and in other classes
```suggestion
    @Override
    public void doWork() {
        System.out.println("Bulldozer starts working.");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer stops.");
    }
```